CMakeLists.txt

```diff
@@ -60,7 +60,7 @@ list(APPEND CMAKE_MODULE_PATH ${LLVM_DIR})
 include(AddLLVM)
 
 set(LLVM_LIT_TOOLS_DIR ${LLVM_INSTALATION_DIR} CACHE STRING "Command used to spawn lit")
-set(LLVM_EXTERNAL_LIT ${LLVM_INTALATION_DIR}/llvm-lit CACHE STRING "Command used to spawn lit")
+set(LLVM_EXTERNAL_LIT ${LLVM_INTALATION_DIR}/llvm-lit)
```

Comment:
Typos here - should be LLVM_INSTALLATION_DIR on this and line 62 (surprised everything still worked despite this!)

It still seems wrong to me. LLVM_LIT_TOOLS_DIR is surely a directory rather than a command so I think the cached string there is the one that needed changing. I can believe that LLVM_EXTERNAL_LIT pointing to llvm-lit executable is a "Command used to spawn lit". What does LLVM_LIT_TOOLS_DIR do? That should be the cached string.

It also seems a shame not to use it in line 63 to define LLVM_EXTERNAL_LIT.

Reply:
Now our regression tests uses LLVM variables. 
LLVM_EXTERNAL_LIT is in cache because LLVM does not cache this variable.